### DISCUSSION TITLE
minor question about copydir log messages

### DISCUSSION THREAD

**Discussion by tinker2much** [gitbls/sdm#191] [General]
I had given these arguments, the only unusual one perhaps being "to=/" : 
```
   * copydir
       Args: from=/home/pi/Documents/sdm/csrc/|to=/|tee=/home/pi/copydir.log

```
And I saw this message in the log:
`Plugin copydir: rsync -a  /home/pi/Documents/sdm/csrc/ /mnt/sdm//`

That double // at the end looks odd, but the plugin seemed to work.

**Comment by gitbls** [comment]
Yea, linux in general seems to treat '//' as '/', but it is annoying.
